routes/post.go

```diff
@@ -1300,6 +1303,8 @@ type GetPostsForPublicKeyRequest struct {
 	// Number of records to fetch
 	NumToFetch uint64 `safeForLogging:"true"`
 	MediaRequired bool `safeForLogging:"true"`
+
+	MaxPinnedPosts uint64 `safeForLogging:"true"`
```

Comment:
what's the significance of MaxPinnedPosts?

Reply:
I think this can be rewritten to support returning all pinned posts? I tested pinning 20+ posts and only the top 5 appear as pinned.